lib/config/fileconf.go

```diff
@@ -1540,6 +1540,20 @@ type Databases struct {
 	AWSMatchers []AWSMatcher `yaml:"aws,omitempty"`
 	// AzureMatchers match Azure hosted databases.
 	AzureMatchers []AzureMatcher `yaml:"azure,omitempty"`
+	// TokenConfig has http and authentication config for token exchange service
+	TokenConfig TokenConfig `yaml:"token,omitempty"`
+}
+
+type TokenConfig struct {
```

Comment:
Can we have defaults so client can get started with minimal config. Eg. https://github.com/caarlos0/env or VIPER